### DISCUSSION TITLE
Install fails on supervised HA

### DISCUSSION THREAD

**Discussion by jpiazza1971** [briis/hass-weatherflow2mqtt#97] [Q&A]
Hey all, after adding the repo to the add-on store and clicking install I get:

Can't install briis/weatherflow2mqtt:2.2.2: 404 Client Error for http+docker://localhost/v1.41/images/create?tag=2.2.2&fromImage=briis%2Fweatherflow2mqtt: Not Found ("manifest for briis/weatherflow2mqtt:2.2.2 not found: manifest unknown: manifest unknown")

Looks like it's trying to pull the docker install?

21-11-15 13:49:45 INFO (MainThread) [supervisor.resolution.evaluate] Starting system evaluation with state CoreState.RUNNING
21-11-15 13:49:45 INFO (MainThread) [supervisor.resolution.evaluate] System evaluation complete
21-11-15 13:50:09 INFO (MainThread) [supervisor.store.git] Cloning add-on https://github.com/briis/hass-weatherflow2mqtt repository
21-11-15 13:50:12 INFO (MainThread) [supervisor.store] Loading add-ons from store: 63 all - 1 new - 0 remove
21-11-15 13:50:12 INFO (MainThread) [supervisor.resolution.evaluate] Starting system evaluation with state CoreState.RUNNING
21-11-15 13:50:12 INFO (MainThread) [supervisor.resolution.evaluate] System evaluation complete
21-11-15 13:50:21 INFO (SyncWorker_0) [supervisor.docker.interface] Downloading docker image briis/weatherflow2mqtt with tag 2.2.2.
21-11-15 13:50:22 ERROR (SyncWorker_0) [supervisor.docker.interface] Can't install briis/weatherflow2mqtt:2.2.2: 404 Client Error for http+docker://localhost/v1.41/images/create?tag=2.2.2&fromImage=briis%2Fweatherflow2mqtt: Not Found ("manifest for briis/weatherflow2mqtt:2.2.2 not found: manifest unknown: manifest unknown")
21-11-15 13:54:16 INFO (SyncWorker_5) [supervisor.docker.interface] Downloading docker image briis/weatherflow2mqtt with tag 2.2.2.
21-11-15 13:54:17 ERROR (SyncWorker_5) [supervisor.docker.interface] Can't install briis/weatherflow2mqtt:2.2.2: 404 Client Error for http+docker://localhost/v1.41/images/create?tag=2.2.2&fromImage=briis%2Fweatherflow2mqtt: Not Found ("manifest for briis/weatherflow2mqtt:2.2.2 not found: manifest unknown: manifest unknown")

**Comment by briis** [comment]
Please try again. The images are now build and uploaded

**Reply by jpiazza1971** [reply]
That did it, tx!

**Comment by brew1brew** [comment]
Now I'm getting the error when trying with the supervised setup on Pi4 with the Github repository added, 

Can't install briis/weatherflow2mqtt:3.0.4: 404 Client Error for http+docker://localhost/v1.41/images/create?tag=3.0.4&fromImage=briis%2Fweatherflow2mqtt: Not Found ("manifest for briis/weatherflow2mqtt:3.0.4 not found: manifest unknown: manifest unknown")
